eventio/tests/test_scripts.py

```diff
@@ -21,3 +38,28 @@ def test_many_object_reprs():
         with EventIOFile(prod2_path) as file_:
             for obj in yield_all_objects_depth_first(file_):
                 assert repr(obj)
+
+
+def test_reprs_prod4():
+    from eventio import EventIOFile
+
+    with EventIOFile(prod4b_astri_file) as f:
+        for i, obj in enumerate(yield_all_objects_depth_first(f)):
+            repr(obj)
```

Comment:
I think `assert repr(obj)` instead of `repr(obj)` catches reprs returning empty strings ..